index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles/mobile.css">
+    <link rel="stylesheet" href="styles/desktop.css">
+    <title>Garvin Chimone</title>
+</head>
+<body>
+    <nav>
+        <div class="title"><h2>My Logo</h2></div>
```

Comment:
- [ ] Kindly consider wrapping your `logo` with an `a` tag to enhance the accessibility and usability of your website. By using an `a` tag, you can transform your logo into a clickable link, providing users with a clear indication that it leads to a specific destination. Additionally, it is a common practice to make logos clickable, allowing users to easily navigate back to the homepage.